src/components/modular-content/iframe-embed/iframe-embed.njk

```diff
@@ -0,0 +1,9 @@
+{% macro iframeEmbed(blockData) %}
+<div class="container">
+  <iframe
+    src="{{ blockData.targetUrl }}"
+    width="100%"
+    height="650"
```

Comment:
How does that work on smaller screens? I'm afraid that 650 can be larger then the window height

Reply:
#mvp